### PR TITLE
Fix for the ConvertProgressEvent

### PR DESCRIPTION
**Pull request by Greek6**
Hi @AydinAdn ,

I'm using the MediaToolKit in my application but the convertProgressEvent isn't working. This should fix it, I hope you can update nuget soon :-)

### CODE REVIEW

**Comment by Temppus** [comment]
Confirm issue.

**Comment by amynkvirani** [comment]
Can the owner merge this to master so it can be used through Nuget as well. Thanks

File: MediaToolkit src/MediaToolkit/Util/RegexEngine.cs [hunk 1]
```diff
@@ -53,22 +53,59 @@ internal static bool IsProgressData(string data, out ConvertProgressEventArgs pr
...
+            double? bitrate = GetDoubleValue(matchBitrate);
 
             progressEventArgs = new ConvertProgressEventArgs(processedDuration, TimeSpan.Zero, frame, fps, sizeKb, bitrate);
 
             return true;
         }
 
+        private static long? GetLongValue(Match match)
+        {
+            try
```
Comment (evandixon) [inline MediaToolkit src/MediaToolkit/Util/RegexEngine.cs]: Try statements are a bit heavy.  I would recommend doing something like this instead:

long parsed;
if (long.TryParse(match.Groups[1].value, out parsed)) {
return parsed;
}
else {
return null;
}

**Comment by tgiachi** [comment]
@AydinAdn  Accept the pull request!! :smile: !

**Comment by AydinAdn** [comment]
Thanks guys, sorry for the slow merge... I'll get NuGet updated tomorrow

**Comment by amynkvirani** [comment]
Hey @AydinAdn were you able to update the NuGet package? On the NuGet website, the last version is from 5 February.

**Comment by baflink** [comment]
Does this resolve this issue?  https://github.com/AydinAdn/MediaToolkit/issues/56

**Comment by Ahmed-Abdelhameed** [comment]
@AydinAdn I believe this wasn't added to the NuGet package.

**Comment by joseluisct** [comment]
Please update the Nuget package, I still have this issue.

**Pull request merged**